expan/core/experiment.py

```diff
@@ -105,6 +108,11 @@ def analyze_statistical_test(self, test, test_method='fixed_horizon', include_da
         treatment_denominators = self._get_denominators(data_for_analysis, test, test.variants.treatment_name)
         treatment_numerators   = treatment * treatment_denominators
 
+        number_of_finite_controls   = np.sum(np.isfinite( control_numerators   / control_denominators   ))
+        number_of_finite_treatments = np.sum(np.isfinite( treatment_numerators / treatment_denominators ))
+
+        if number_of_finite_controls < 2 or number_of_finite_treatments < 2: return test_result
```

Comment:
Is there maybe a better alternative than returning an empty result?

Reply:
That seems to be the convention in this function; I'm just mirroring the behaviour of `_is_valid_for_analysis`. But yes, I must I don't really know what the conventions are in Expan about what to return and when

Reply:
It might be better to move this logic into `_is_valid_for_analysis`